index.html

```diff
@@ -290,6 +290,18 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    
+    <div class="footer">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mvodvrnv" method="post" class="form">
+        <input type="text" placeholder="Full name" name="name" required>
```

Comment:
- [ ] According to the requirements, the name input should have a `maxlength="30"` attribute. So it will be good to add that.